### DISCUSSION TITLE
Long used regex is suddenly invalid?

### DISCUSSION THREAD

**Discussion by nielsbasjes** [renovatebot/renovate#21857] [Request Help] [answered]
Since yesterday just about all my projects  (using Github app, which all have similar renovate.json files) started to fail with this error.
 ```
Location: renovate.json
Error type: The renovate configuration file contains some invalid settings
Message: Invalid regExp for packageRules[0].allowedVersions: '!/^(?i).*[-_\.](Alpha|Beta|RC|M|EA|Snap|snapshot|jboss|atlassian)[-_\.]?[0-9]?.*$/'
```

The underlying config for these projects has not changed for many months and worked on all of these projects fine.

Seem like something changed in Renovate. Is this a bug in Renovate or must I change something?

https://github.com/nielsbasjes/prefixmap/issues/164
https://github.com/nielsbasjes/yauaa/issues/255

**Comment by rarkins** [comment, marked as answer]
Thank you for reporting this. It was a strange error - caused by our regex library being installed into an older version of node than currently running. I have refreshed all backend servers and the issue has been autoclosed in the first of our repos I tried, so I'm pretty sure it's solved

**Reply by nielsbasjes** [reply]
Thanks.
I can confirm the issue is being closed on my projects too.

**Reply by nielsbasjes** [reply]
@rarkins Last qiuestion about this: Was this a "Github app" specific problem or was this also a problem in the docker image (i.e. will it affect self hosted situations)?

**Reply by rarkins** [reply]
It was the GitHub app only.
